target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -426,6 +426,18 @@ define Device/tplink_eap245-v3
 endef
 TARGET_DEVICES += tplink_eap245-v3
 
+define Device/tplink_ec230-g1
+  $(Device/tplink-v2)
+  SOC := qca9563
+  IMAGE_SIZE := 16000k
+  DEVICE_MODEL := EC230-G1
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca9888-ct
+  TPLINK_HWID := 0x4E8C8701
+  TPLINK_HWREV := 0x00060034
+  TPLINK_FLASHLAYOUT := 16Mqca
```

Comment:
Where do these three come from?

Reply:
From the original image header, parsed [here (gist)](https://gist.github.com/izuannazrin/0c32f269b7ff008db04ad0403a22acc0). Though from my testing it's not necessary but for the sake of correctness, \*insert idk emoji\* why not